.github/workflows/release.yml

```diff
@@ -0,0 +1,151 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+---
+name: terraform-boot-cd
+
+on:
+  workflow_dispatch:
+    inputs:
+      ReleaseType:
+        type: choice
+        description: Select the next development version
+        options:
+          - Major Version
+          - Minor Version
+          - Patch Version
+
+env:
+  BOT_USER_NAME: eclipse-terraform-boot-bot
+  BOT_EMAIL_ID: terraform-boot-bot@eclipse.org
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'eclipse-xpanse/terraform-boot'
+    outputs:
+      next-version: ${{ steps.new_version.outputs.next-version }}
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+
+      - name: Set Up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          gpg-private-key: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
```

Comment:
these lines upto line 47 can be removed.

Reply:
fixed